### PR TITLE
Add Timber logging library

### PR DESCRIPTION
**Pull request by jsulli**
Included Crashlytics tree.

### CODE REVIEW

File: Awful.apk/src/main/java/com/ferg/awfulapp/CrashlyticsReportingTree.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.ferg.awfulapp
+
+import com.crashlytics.android.Crashlytics
+import timber.log.Timber
+
+
+class CrashlyticsReportingTree : Timber.Tree() {
+    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
```
Comment (Sereri) [inline Awful.apk/src/main/java/com/ferg/awfulapp/CrashlyticsReportingTree.kt]: Can you please improve the variable name of "t"?

File: Awful.apk/src/main/java/com/ferg/awfulapp/AwfulApplication.java [hunk 2]
```diff
@@ -52,6 +51,8 @@ public void onCreate() {
 		AnnouncementsManager.init();
         onPreferenceChange(mPref,null);
 
+
```
Comment (Sereri) [inline Awful.apk/src/main/java/com/ferg/awfulapp/AwfulApplication.java]: Are these necessary?

File: Awful.apk/src/main/java/com/ferg/awfulapp/AwfulApplication.java [hunk 3]
```diff
@@ -61,19 +62,23 @@ public void onCreate() {
 		} catch (PackageManager.NameNotFoundException e) {
 			e.printStackTrace();
 		}
-		Log.i(TAG, String.format("App installed %d hours ago", hoursSinceInstall));
+		Timber.i("App installed %d hours ago", hoursSinceInstall);
+
```
Comment (Sereri) [inline Awful.apk/src/main/java/com/ferg/awfulapp/AwfulApplication.java]: Same about this empty line

**Comment by jsulli** [comment]
Old habits, sorry. Changes made.

**Pull request merged**